.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,7 @@
-Thank you for your contribution! If you have any questions about your PR,
-or need help completing it, you can ping the maintainers of this repository
-who will be happy to help if they can find time.
-You can optionally use the following check list when you work on your PR:
+Thank you for your contribution! 
+If you have any questions about your PR, or need help completing it, you can ping the maintainers of this repository, who will be happy to help if they can find time.
```

Comment:
I think I prefer the version with semantic line breaks here.

Reply:
I was prompted to make this PR because, in practice, the template looked odd with the line breaks (see screenshot of the current template). These lines also wrap in unexpected ways when resizing the window.

<img width="917" alt="image" src="https://github.com/BioJulia/Contributing/assets/24236593/cc5bb0e9-46be-4773-ac85-80a0336b606d">

Reply:
Oh, that's bizzare - seems like github doesn't treat linebreaks
as
spaces :sob:

Reply:
The PR also groups the list lead with the list items.